pkg/api/index.go

```diff
@@ -165,6 +165,12 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool) ([]*dto
 			Url:  hs.Cfg.AppSubURL + "/dashboard/snapshots",
 			Icon: "camera",
 		})
+		dashboardChildNavs = append(dashboardChildNavs, &dtos.NavLink{
+			Text: "Global panels",
+			Id:   "library-panels",
+			Url:  hs.Cfg.AppSubURL + "/library-panels",
+			Icon: "reusable-panel",
```

Comment:
Oh no! We use 3 different names for the same thingy in one place! 🙈

Reply:
Hehe, yes but we're keeping the UI consistent at least. This is a tricky one that will follow us until v8 I think. Everything in the backend is called library panels including tables in db. But I agree this is not optimal.